.travis.yml

```diff
@@ -56,10 +61,21 @@ install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install geojson Shapely sphinx sphinx_bootstrap_theme sphinx-intl odoo-sphinx-autodoc

```

Comment:
This is not needed - some packages are already on line 57 and sphinx is used in OCA but not in our repositories